packages/react-app/src/components/Modals/Wrap.jsx

```diff
@@ -19,26 +20,29 @@ const Wrap = ({ contractLoader, accountAddress, vestedId, writeContracts, tx, cl
   );
 
   const handleWrap = useCallback(async () => {
+    const amount = BigNumber.from(toDecimals(state.underlyingAmount, 18));
```

Comment:
I reuse the math util we had on Gardens to transform an input amount into a string decimal that we can safely use with BigNumbers.